src/60_Samples_%26_Templates/Hotel.html

```diff
@@ -1,11 +1,12 @@
 <!---{
   "preview": "default",
-  "draft": true
+  "draft": true,
+  "experiments": ["amp-date-picker", "amp-lightbox-gallery"]
   }--->
   <!--
   ## Introduction
 
-  This is a sample template for a hotel listing in AMP.
+  This is a sample showing how to implement an hotel page.
```

Comment:
Can you add a few more details about this sample, in particular which use cases we are covering.

Reply:
done